### DISCUSSION TITLE
Feature Request: Search embedded PDFs as if they were part of the parent note

### DISCUSSION THREAD

**Discussion by sbj175** [scambier/obsidian-omnisearch#283] [Ideas]
Would it be possible to search "searchable" PDFs that are *embedded/linked* to a note? Currently, there is no way of combining a tag search, which can only get a hit in the parent markdown document, with a PDF content search, which can only get a hit in the attached (embedded/linked) PDF.

This capability would enable a search such as "#myTag pdfText" to find results where this is not possible today.

**Comment by sbj175** [comment]
I'm surprised this hasn't gotten any traction. In my opinion this is an Evernote-killer type of feature. Does it need better explanation?

**Reply by scambier** [reply]
Discussions don't have much visibility (the forum isn't really active), so this feature request should be better in "Issues".

Anyway it's a great idea, I just personally haven't had time to work on it (and pull requests are welcome ^^)

Related #245

**Reply by sbj175** [reply]
I appreciate that. I was a full time Java developer years ago. Never learned Javascript or Typescript, but I just may give it a go! Thanks.
